.circleci/workflows.yml

```diff
@@ -28,7 +28,8 @@ mainBuildFilters: &mainBuildFilters
       only:
         - develop
         - /^release\/\d+\.\d+\.\d+$/
-        - 'mikep/22825-next-middleware'
+        # use the following branch as well to ensure that v8 snapshot cache updates are fully tested
+        - 'update-v8-snapshot-cache-on-develop'
```

Comment:
Do we need to update line 137 to always persist and run the full binary tests on this branch? https://github.com/cypress-io/cypress/pull/25784/files#diff-7bf091450b804ea25dc3d11ac28c50202d19178f7b53f54a02ad59c946afa6a7R137

Reply:
Updated: [`4fcc2ef` (#25784)](https://github.com/cypress-io/cypress/pull/25784/commits/4fcc2efb679babf127c1c8dbab99903436d48e54)